### PR TITLE
2 create counter atom

### PR DESCRIPTION
**Pull request by ArturAssisComp**

### CODE REVIEW

File: crdt/ctree.go [hunk 1]
```diff
@@ -870,6 +871,17 @@ func (t *CausalTree) SetCursor(i int) error {
 	return nil
 }
 
+// +--------------------------------------+
+// + Operations - Atom Priority constants |
+// +--------------------------------------+
+const (
+	InsertCharPriority    = 0
```
Comment (brunokim) [inline crdt/ctree.go:878]: These constants don't need to be exported, use minuscule for the first letter

File: crdt/testdata/fuzz/FuzzList/ctrlaltdel [hunk 2]
```diff
@@ -1,3 +0,0 @@
-go test fuzz v1
```
Comment (brunokim) [inline crdt/testdata/fuzz/FuzzList/ctrlaltdel:1]: Please revert this deletion

File: crdt/testdata/fuzz/FuzzList/xabdy [hunk 3]
```diff
@@ -1,3 +0,0 @@
-go test fuzz v1
```
Comment (brunokim) [inline crdt/testdata/fuzz/FuzzList/xabdy:1]: Also this

File: crdt/testdata/fuzz/FuzzList/ctrlaltdel (hunk 2 above)
Comment (ArturAssisComp) [reply crdt/testdata/fuzz/FuzzList/ctrlaltdel]: Reverted.

File: crdt/testdata/fuzz/FuzzList/xabdy (hunk 3 above)
Comment (ArturAssisComp) [reply crdt/testdata/fuzz/FuzzList/xabdy]: Reverted.

File: crdt/ctree_test.go [hunk 4]
```diff
@@ -495,21 +495,128 @@ func TestMergeMultipleStrContainers(t *testing.T) {
...
+			{op: checkJSON, local: 0, str: `[12]`},
+			// Insert counter 23 into site 1
+			{op: insertCounter, local: 1},
+			{op: insertAdd, local: 1, val: 23},
+			{op: checkJSON, local: 1, str: `[23]`},
+			// Merge site #1 -> site #0
+			{op: merge, local: 0, remote: 1},
+			{op: checkJSON, local: 0, str: `[12, 23]`},
+		})
+	})
```
[the remark this answers is not in the document]
Comment (ArturAssisComp) [reply crdt/ctree_test.go:544]: Test case added.

File: crdt/ctree.go (hunk 1 above)
Comment (ArturAssisComp) [reply crdt/ctree.go]: Done.

**Review by brunokim — APPROVED** [review]
Great work!

**Pull request merged**